doc/SpectralInstrument/spectral.rst

```diff
@@ -12,10 +12,10 @@ Properties
 ====================================== ========================= ================== ===============================================
 Property name                          Default value             Type               Description
 ====================================== ========================= ================== ===============================================
-ExpertConnectionAddress                127.0.0.1                 String             Server name or IP address of the SI Image SGL II software
-ExpertConnectionPort                   0                         String             TCP/IP port of the SI Image SGL II software
-ExpertImagePacketPixelsNb              512                       Long               Number of pixels sent into a image part TCP/IP packet
-ExpertImagePacketDelayMicroSec         300                       Long               Delay between the sending of two image part TCP/IP packets (in micro-seconds)
+ConnectionAddress                      127.0.0.1                 String             Server name or IP address of the SI Image SGL II software
+ConnectionPort                         0                         String             TCP/IP port of the SI Image SGL II software

```

Comment:
rename all the "Spectral" into "SpectralInstrument" in this file as well as the name the file itself